server/lib/configuration.js

```diff
@@ -340,8 +318,18 @@ const conf = module.exports = convict({
     }
   },
   marketing_email: {
+    api_key: {
+      default: 'test key please change',
+      doc: 'Basket API key',
+      format: String
+    },
+    api_timeout: {
+      default: '5 seconds',
+      doc: 'Timeout for talking to the Basket API server, in ms',
+      format: Number
```

Comment:
Should this be `format: 'duration'` ?

Reply:
I just copied this these over, but that seems like a fine suggestion while I'm in here making updates.